.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Should JSON files have comments in?

Reply:
This is generated by VS Code, but i've removed it anyway.